src/content/docs/en/pages/guides/marketplace/templates/hello-world/hello-world.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: How to deploy the Hello World template
+description: With this template, deploy an Hello World application to test edge computing capabilities.
+meta_tags: templates, guides, Azion Marketplace
+namespace: docs_guides_hello_world_template
+permalink: /documentation/products/guides/hello-world-template/
+---
+
+import Badge from '~/components/Badge.astro';
+
+<Badge variant="accent">
+Preview
+</Badge>
+
+The Azion **Hello World** template deploys an edge application that prints a *Hello, World!* message in your browser and you can use it as a starting point to become familiar with **Azion Edge Platform** and edge computing capabilities.
+
+---
+
+## Getting the template
+
+To use the **Hello World** template, access [Real-Time Manager (RTM)](https://manager.azion.com/) and select the **Start with a template** option on the homepage.
+
+This will open the building page where you can select the **Hello World** card.
```

Comment:
É building page mesmo? Start the application building process?

Reply:
ajustei aqui para ser mais genérico